recipe/meta.yaml

```
@@ -0,0 +1,94 @@
+{% set name = "google-genai" %}
+{% set version = "1.38.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/googleapis/python-genai/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: da73c69fc8d42152fab7cfb89c30f13b9379e8d15b131c1feb8675e10c302585
+  patches:            # [py<311]
+    - fix_tests.patch # [py<311]
+
+build:
+  number: 0
+  skip: True # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - wheel
+    - twine >=6.1.0
+    - packaging >=24.2
+    - pkginfo >=1.12.0
+    - pip
+  run:
+    - python
+    - anyio >=4.8.0,<5.0.0
+    - google-auth >=2.14.1,<3.0.0
+    - httpx >=0.28.1,<1.0.0
+    - pydantic >=2.0.0,<3.0.0
+    - requests >=2.28.1,<3.0.0
+    - tenacity >=8.2.3,<9.2.0
+    - websockets >=13.0.0,<15.1.0
+    - typing_extensions >=4.11.0,<5.0.0
```

https://github.com/googleapis/python-genai/blob/v1.38.0/pyproject.toml#L40
```suggestion
    - typing_extensions >=4.11.0,<5.0.0
  run_constrained:
    # aiohttp
    - aiohttp <4.0.0
    # local-tokenizer
    - sentencepiece >=0.2.0
```